configs/config.toml.SAMPLE

```diff
@@ -86,6 +86,14 @@
 # This is an optional parameter, if not provided timeout for TSS protocol
 # pre-parameters generation will be set to `2 minutes`.
 #  PreParamsGenerationTimeout = "2m30s"
+#
+# TSS pre-parameters target pool size. This value determines the number of
+# pre-parameters that should be generated and kept at hand ready to use.
+# That value should be set reasonably because a low pool size may force the
+# client to generate parameters during protocol executions and cause unwanted
+# delays. On the other hand, a big pool size can cause high CPU usage for
+# a long time. The default value of this parameter is `20`.
+#  PreParamsTargetPoolSize = 20
```

Comment:
Why not just `PreParamsPoolSize`? Is the `*Target*` part needed here? It's a pool, we don't guarantee it's always at the requested level.

Reply:
> It's a pool, we don't guarantee it's always at the requested level.

Exactly, this is why this parameter is the `target` the pool is aiming for. Also, `PreParamsPoolSize` is rather the current size and it's exposed as such in the metrics API: https://github.com/keep-network/keep-ecdsa/pull/618

Reply:
Yeah, I've just finished the review of #618 and I now understand why you added `*Target*` here. Resolving this conversation.

Reply:
I'm wondering about naming it `PreParamsPoolSizeTarget`... Would be helpful e.g. when greping the code for `PreParamsPoolSize` this parameter would also be found.